yt/java/ytsaurus-client/src/main/java/tech/ytsaurus/client/ApiServiceClientImpl.java

```diff
@@ -905,6 +905,9 @@ public CompletableFuture<Void> alterTableReplica(AlterTableReplica req) {
 
     @Override
     public CompletableFuture<GUID> startOperation(StartOperation req) {
+        var spec = req.getSpec();
+        StartOperationHelpers.guessBaseLayers(spec);
```

Comment:
Let's move this to [MapperOrReducerSpec.prepare](https://github.com/ytsaurus/ytsaurus/blob/main/yt/java/ytsaurus-client/src/main/java/tech/ytsaurus/client/operations/MapperOrReducerSpec.java#L198).